src/lib/Structures/Base/SocketHandler.ts

```diff
@@ -104,8 +104,24 @@ export abstract class SocketHandler {
 
 }
 
+/**
+ * A raw message
+ * @since 0.5.0
+ */
```

Comment:
This is private, I forgot to mark it as such:
```suggestion
 * @private
 */
```